packages/itwin/ec3-widget/package.json

```diff
@@ -0,0 +1,114 @@
+    "copy:assets": "npm run -s copy:public && npm run -s copy:cjs && npm run -s copy:esm",
+    "copy:cjs": "cpx \"./src/**/*.scss\" ./lib/cjs",
+    "copy:esm": "cpx \"./src/**/*.scss\" ./lib/esm",
+    "copy:public": "cpx \"./public/**/*\" ./lib/public",
+    "cover": "npm test -- --coverage",
+    "dual-build": "npm run -s build:cjs && npm run -s build:esm",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "lint:fix": "npm run lint -- --fix",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./public/locales/en-PSEUDO",
+    "rebuild": "npm run clean && npm run build",
+    "test": "jest",
+    "test:watch": "jest --watch "
+  },
+  "eslintConfig": {
+    "plugins": [
+      "@itwin"
+    ],
+    "extends": "plugin:@itwin/ui",
+    "rules": {
+      "no-duplicate-imports": "off",
+      "@typescript-eslint/consistent-type-imports": "error",
+      "deprecation/deprecation": "off"
+    }
+  },
+  "dependencies": {
+    "@itwin/imodels-access-frontend": "^1.0.1",
+    "@itwin/imodels-client-management": "^1.0.1",
+    "@itwin/insights-client": "^0.2.0",
+    "@itwin/itwinui-icons-color-react": "^1.0.1",
+    "@itwin/itwinui-icons-react": "^1.7.0",
+    "@itwin/itwinui-react": "~1.42.0",
+    "react-redux": "^8.0.2",
+    "simple-react-validator": "^1.6.1"
+  },
+  "devDependencies": {
+    "@faker-js/faker": "^6.3.1",
+    "@itwin/appui-abstract": "^3.0.0",
+    "@itwin/appui-layout-react": "^3.0.0",
+    "@itwin/appui-react": "^3.0.0",
+    "@itwin/build-tools": "^3.0.0",
+    "@itwin/components-react": "^3.0.0",
+    "@itwin/core-bentley": "^3.0.0",
+    "@itwin/core-common": "^3.0.0",
+    "@itwin/core-frontend": "^3.0.0",
+    "@itwin/core-geometry": "^3.0.0",
+    "@itwin/core-markup": "^3.0.0",
+    "@itwin/core-orbitgt": "^3.0.0",
+    "@itwin/core-quantity": "^3.0.0",
+    "@itwin/core-react": "^3.0.0",
+    "@itwin/eslint-plugin": "^3.0.0",
+    "@itwin/imodel-components-react": "^3.0.0",
+    "@itwin/itwinui-css": "^0.50.0",
+    "@itwin/presentation-common": "^3.0.0",
+    "@itwin/presentation-frontend": "^3.0.0",
+    "@itwin/webgl-compatibility": "^3.0.0",
+    "@testing-library/dom": "^8.12.0",
+    "@testing-library/jest-dom": "^5.16.3",
+    "@testing-library/react": "^12.1.4",
+    "@testing-library/user-event": "^14.1.1",
+    "@types/i18next": "^8.4.2",
+    "@types/jest": "^27.5.2",
+    "@types/node": "^14.14.20",
+    "@types/react": "^18.0.15",
+    "@types/react-dom": "^18.0.6",
+    "@types/react-table": "^7.7.0",
+    "@types/testing-library__jest-dom": "^5.14.3",
+    "@typescript-eslint/eslint-plugin": "^5.3.1",
+    "@typescript-eslint/parser": "^5.10.1",
+    "cpx2": "^3.0.0",
+    "eslint": "^7.32.0",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-react": "^7.25.1",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "jest": "^27.5.1",
+    "jest-cli": "^27.5.1",
+    "jest-transform-stub": "^2.0.0",
+    "msw": "^0.39.2",
+    "react": "^18.2.0",
```

Comment:
Downgrade to what is required from peerDependencies. This will also make it consistent with other packages.